src/picasso-components/custom-rect/index.js

```diff
@@ -0,0 +1,27 @@
+export default {
+  require: ['renderer'],
+  defaultSettings: {},
+  renderer: 'dom',
+  render(h) {
+    if (!this.settings.show()) {
```

Comment:
This should not be needed, as `render` should not be called if show resolves to false.

Reply:
you are absolutely right.